### PR TITLE
Add `pgroll` `v0.8` blog post

### PR DESCRIPTION
**Pull request by andrew-farries**
## Summary

Add a blog post covering what's new in the upcoming`pgroll` `v0.8.0` release.

The post re-uses the same OG image as the `v0.7` and `v0.6` posts.

Direct link to post: https://xata-blog-pr-338.vercel.app/blog/pgroll-0-8-0-update

### Blog Release Checklist

- \[ ] Release on: \<mm/dd/yy>
- \[ ] Published on dev.to (connected to Xata org)
- \[ ] OG image included
- \[ ] Social media is scheduled for this post

### CODE REVIEW

File: whats-new-in-pgroll-0-8-0.mdx [hunk 1]
```diff
@@ -0,0 +1,396 @@
+---
+title: pgroll 0.8.0 update
+description: Learn about the latest changes in pgroll in the v0.8.0 release as we add new CLI commands, expand what's possible with multi-column keys and constraints and add performance benchmarks
```
Comment (CezzaineZ) [inline whats-new-in-pgroll-0-8-0.mdx:3]:
```suggestion
description: Learn about the latest changes in pgroll in the v0.8.0 release as we add new CLI commands, expand what's possible with multi-column keys and constraints and add performance benchmarks.
```

File: whats-new-in-pgroll-0-8-0.mdx [hunk 2]
```diff
@@ -0,0 +1,396 @@
...
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/pgroll-update-0-6-0@2x.jpg
+  alt: pgroll
+author: Andrew Farries
+authorEmail: andrew@xata.io
+date: 11-28-2024
```
Comment (CezzaineZ) [inline whats-new-in-pgroll-0-8-0.mdx:9]:
```suggestion
date: 12-09-2024
```

File: whats-new-in-pgroll-0-8-0.mdx [hunk 3]
```diff
@@ -0,0 +1,396 @@
...
+- The duplicated columns are backfilled; data is copied from the original columns and rewritten using the `up` SQL expression for the column from the migration file.
+
+At this point the migration is active and client applications can access the version of the schema with or without the new constraint by setting the Postgres search path as described in the migration [workflow section](https://github.com/xataio/pgroll/blob/main/docs/getting-started.md#migration-workflow) of the pgroll documentation.
+
+As long as the migration is active:
+
+- records written by clients accessing the new version of the schema with the constraint are dual-written to the version without the constraint using the `down` SQL expressions from the migration definition.
+- records written by clients accessing the old version of the schema without the constraint are dual-written to the version with the constraint using the `up` SQL expressions from the migration definition.
+
+On migration completion, the original columns are dropped and the duplicated ones renamed to take their place.
```
Comment (CezzaineZ) [inline whats-new-in-pgroll-0-8-0.mdx:168]:
```suggestion
On migration completion, the original columns are dropped and the duplicated ones are renamed to take their place.
```

File: whats-new-in-pgroll-0-8-0.mdx [hunk 4]
```diff
@@ -0,0 +1,396 @@
...
+06_add_column_to_sql_table.json
+```
+
+Running `pgroll migrate` again will run only the new migrations that have not yet been applied:
+
+```bash
+$ pgroll migrate migrations/ --complete
+```
+
+The applies the new migrations to the target database schema:
```
Comment (CezzaineZ) [inline whats-new-in-pgroll-0-8-0.mdx:288]:
```suggestion
Then applies the new migrations to the target database schema:
```

File: whats-new-in-pgroll-0-8-0.mdx [hunk 5]
```diff
@@ -0,0 +1,396 @@
...
+
+```bash
+SUCCESS  New version of the schema available under the postgres "public_05_sql" schema
+SUCCESS  New version of the schema available under the postgres "public_06_add_column_to_sql_table" schema
+```
+
+All migrations applied by the `pgroll migrate` command are completed except the final one which is started and left active. To also complete fhe final migration, use the `--complete` flag to `pgroll migrate`.
```
Comment (CezzaineZ) [inline whats-new-in-pgroll-0-8-0.mdx:295]:
```suggestion
All migrations applied by the `pgroll migrate` command are completed except the final one which is started and left active. To also complete the final migration, use the `--complete` flag to `pgroll migrate`.
```

**Review by CezzaineZ — COMMENTED** [review]
Great blog @andrew-farries! Just spotted a few typos. The other thing worth noting is that when we move over the docs to pgroll.com we'll need to update all references to the docs for this blog and all the prior ones as well.

File: whats-new-in-pgroll-0-8-0.mdx (hunk 4 above)
Comment (andrew-farries) [reply whats-new-in-pgroll-0-8-0.mdx]: It should say 'This applies...'. Fixed

**Comment by andrew-farries** [comment]
Thanks for the review @CezzaineZ 🙏 I've applied all the fixes.

File: whats-new-in-pgroll-0-8-0.mdx [hunk 6]
```diff
@@ -0,0 +1,396 @@
...
+image:
+  src: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/pgroll-update-0-6-0@2x.jpg
+  alt: pgroll
+author: Andrew Farries
+authorEmail: andrew@xata.io
+date: 12-09-2024
+tags: ['open-source', 'postgres', 'schema', 'migrations', 'pgroll', 'fpSchemaMigrations', 'oss']
+published: true
+slug: pgroll-0-8-0-update
+ogImage: https://raw.githubusercontent.com/xataio/mdx-blog/main/images/pgroll-update-0-6-0@2x.jpg
```
Comment (tsg) [inline whats-new-in-pgroll-0-8-0.mdx:13]: Add a line like this so we don't get penalized for duplicate content:

```
canonicalUrl: https://pgroll.com/blog/pgroll-080-update
```

**Review by CezzaineZ — APPROVED** [review]
LGTM!

**Pull request merged**